package.json

```diff
@@ -243,6 +248,7 @@
         "query-string": "5.1.1",
         "react": "^18.3.1",
         "react-animate-height": "^3.2.3",
+        "react-aria-components": "^1.10.1",
```

Comment:
blueprint already installs react-aria-components as a regular dependency. do we still need to add it? why was it not required before?

Reply:
using it directly now, for types, previously we werent